pulumi/aws/start_all.sh

```diff
@@ -35,9 +39,29 @@ if ! grep --quiet '^PULUMI_STACK=.*' "${script_dir}/config/environment"; then
   echo "PULUMI_STACK=${PULUMI_STACK}" >> "${script_dir}/config/environment"
 fi
 
+if ! grep --quiet '^AWS_DEFAULT_REGION=.*' "${script_dir}/config/environment"; then
+  read -r -e -p "Enter the name of the AWS Region to use in all projects: " AWS_DEFAULT_REGION
+  echo "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" >> "${script_dir}/config/environment"
+fi
+
+if ! grep --quiet '^AWS_DEFAULT_PROFILE=.*' "${script_dir}/config/environment"; then
```

Comment:
If `AWS_PROFILE` is set, then pulumi will pull the profile name from the environment. We do not want to interrupt that mode of operation. Perhaps we test for the presence of that envvar here and at line 62.

Reply:
There is also the potential for the user to not have a profile defined and be using only the default credentials.

Reply:
That's a good point; I'll get that change in.